README.md

```diff
@@ -74,6 +74,11 @@ $ mvn clean install -Dit.test=RtGistITCase -Dfailsafe.github.key=<token> -Dfails
 
 Replace `<token>` with the OAuth access token, and `<repo>` with the name of
 repository you create in your account (for test purposes only), for example `yegor256/test`.
+The failsafe.github.key should have permissions in failsafe.github.repo to all scopes needed
```

Comment:
Just one more thing - let's mark out `failsafe.github.key` and `failsafe.github.repo` as code, to be consistent with the rest of the document. You can do that by using the ` character in the Markdown code:

```markdown
The `failsafe.github.key` should have permissions in `failsafe.github.repo` to all scopes needed
```